app/dashboard/models.py

```diff
@@ -764,3 +764,21 @@ def normalize_tip_usernames(sender, instance, **kwargs):
 
 
 m2m_changed.connect(m2m_changed_interested, sender=Bounty.interested.through)
+
+
+class CoinRedemption(SuperModel):
+  shortcode = models.CharField(max_length=255, default='')
+  url = models.URLField(null=True)
+  network = models.CharField(max_length=255, default='')
+  token_name = models.CharField(max_length=255)
+  contract_address = models.CharField(max_length=255)
+  amount = models.IntegerField(default=1)
+  expires_date = models.DateTimeField()
```

Comment:
Per our slack discussion, do you think you can either override `save()` on this model and `self.contract_address = web3.toChecksumAddress(self.contract_address)` or perform the `toChecksumAddress` in the management command to ensure the contract_address is always stored such that it'll succeed during redemption?

Reply:
Disregard this, I'll update it after this is merged.